crates/sui-source-validation-service/src/lib.rs

```diff
@@ -54,6 +59,9 @@ pub const LOCALNET_WS_URL: &str = "ws://127.0.0.1:9000";
 
 pub const WS_PING_INTERVAL: Duration = Duration::from_millis(20_000);
 
+pub const METRICS_ROUTE: &str = "/metrics";
+pub const METRICS_HOST_PORT: &str = "127.0.0.1:9814";
```

Comment:
erm. I have found evidence _handwave_ that exposing `:9814/metrics` is one possible port to use for the prom server? @after-ephemera what's an acceptable value here (and can I hardcode it?)

Reply:
Yeah [we default to 9184](https://github.com/MystenLabs/sui-operations/blob/main/pulumi/common/mysten_service.py#L121) and [`/metrics`](https://github.com/MystenLabs/sui-operations/blob/main/pulumi/common/mysten_service.py#L203) so it's fine to default to these and hardcode them.